product-service/handlers/getProduct.ts

```diff
@@ -0,0 +1,31 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import 'source-map-support/register';
+import productList from '../productList.json';
+
+export const getProduct: APIGatewayProxyHandler = async (event, _context) => {
+	const productId = event.pathParameters?.productId;
+	const product = productList.find(
+		({ id }) => productId === id
+	);
+
+	if (product !== null && product !== undefined) {
+		return {
+			statusCode: 200,
+			headers: {
+				'Access-Control-Allow-Headers': 'Content-Type',
+				'Access-Control-Allow-Origin': 'https://d2tvnqhil9jwg2.cloudfront.net',
```

Comment:
In case you start adding another consumer, you should add it here as well, so potentially in future it will be a boring routine to handle